client/src/pages/HomePage.jsx

```diff
@@ -1,26 +1,93 @@
 import React from "react";
-import { Grid, Stack } from "@mui/material";
+import { Grid, Stack, Typography, Box, useTheme } from "@mui/material";
 
 import BasePage from "./BasePage";
 import BackgroundShapes from "../components/BackgroundShapes";
 import HomePageContent from "../components/HomePageContent";
 import SearchBar from "../components/SearchBar";
 
-/**
- * https://github.com/BobaTalks/internship-tracker/issues/24
- * Will need to include the background styles for the HomePage per design spec
- */
+import metaLogo from "../assets/meta_logo.svg";
+import microsoftLogo from "../assets/microsoft_logo.svg";
+import amazonLogo from "../assets/amazon_logo.svg";
+import appleLogo from "../assets/apple_logo.svg";
+import googleLogo from "../assets/google_logo.svg";
+
+const companyURLs = [
+  microsoftLogo,
+  metaLogo,
+  amazonLogo,
+  googleLogo,
+  appleLogo,
+];
+
 const HomePage = (props) => {
+  const theme = useTheme();
+
   return (
     <BasePage>
       <BackgroundShapes />
-      <Stack spacing={5} sx={{ zIndex: 0 }} minWidth="100%">
+      <Stack
+        spacing={5}
+        sx={{ zIndex: 0, marginBottom: "30px" }}
+        minWidth="100%"
+      >
         <HomePageContent />
         <Grid container justifyContent="center">
           <Grid item lg={8} md={9} xs={11}>
             <SearchBar isNarrow />
           </Grid>
         </Grid>
+        <Typography
+          sx={{
+            textAlign: "center",
+            fontWeight: "1000",
```

Comment:
Do we have a preset or typography heading we can use here that already has achieved the styling we need for this?

Reply:
Yes we do! Just changed it to use the h6 variant